plugin/harness/execer.go

```diff
@@ -138,18 +153,35 @@ func (e *Execer) Exec(ctx context.Context) error {
 
 		// execute the binary
 		cmd := exec.Command(shell, path)
-		cmd.Env = e.Environ
-		cmd.Dir = e.Workdir
-		cmd.Stderr = e.Stderr
-		cmd.Stdout = e.Stdout
-		if err := cmd.Run(); err != nil {
+		err = runCmds(ctx, []*exec.Cmd{cmd}, e.Environ, e.Workdir, e.Stdout, e.Stderr)
+		if err != nil {
 			return err
 		}
 	}
 
 	return nil
 }
 
+func (e *Execer) buildGoExecutable(ctx context.Context, module string) (
+	string, error) {
+	defer timer("buildGoExecutable")()
```

Comment:
`if debug: defer ...` right?

Reply:
timer method only logs if debug is enabled.